src/views/ProfileSettings/cards/ResearchAndUsage/index.tsx

```diff
@@ -24,7 +24,9 @@ const initialChangedValues = {
 };
 
 const hasOtherUsage = (userUsages: string[]) =>
-  userUsages.find((usage) => !usageOptions.find((defaultUsage) => defaultUsage.value === usage));
+  userUsages.find(
```

Comment:
i would use `some` and something like (to be tested!):

`userUsages.some(u => !interestsOptions.map(x => x.value).includes(u))`

micro-optim: would be to compute `interestsOptions.map(x => x.value)` first

Reply:
I will use `some` but I prefer to keep filter in the loop, it's easier to read that a `map` and a `includes`